src/Goerli-DssSpell.sol

```diff
@@ -24,6 +24,10 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralOnboardingAction } from "./Goerli-DssSpellCollateralOnboarding.sol";
 
+import "dss-interfaces/dss/EndAbstract.sol";
```

Comment:
Can comment out the ABIEncoderV2 line (a few lines above).

Reply:
Done.